ztunnel-1.24.yaml

```diff
@@ -0,0 +1,78 @@
+package:
+  name: ztunnel-1.24
+  version: 1.24.0
+  epoch: 0
+  description: The `ztunnel` component of istio ambient mesh.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - ztunnel=${{package.full-version}}
+    runtime:
+      - ca-certificates-bundle
+      - glibc
+      - ip6tables
+      - iptables
+      - libgcc
+      - libmnl
+      - libnetfilter_conntrack
+      - libnfnetlink
+      - wolfi-baselayout
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - cargo-auditable
+      - clang
+      - cmake
+      - openssf-compiler-options
+      - protoc
+      - rust
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/istio/ztunnel
+      tag: ${{package.version}}
+      expected-commit: 1226c1b35f50938f428c71f7dcad3602ea991675
+
+  - runs: |
+      export ZTUNNEL_BUILD_buildStatus=clean
+      export ZTUNNEL_BUILD_buildGitRevision=$(git rev-parse HEAD)
+      export ZTUNNEL_BUILD_buildTag=$(git describe --tags --abbrev=0)
+      export ZTUNNEL_BUILD_buildVersion=$(git rev-parse HEAD)
+       cargo auditable build --release
+       mkdir -p ${{targets.destdir}}/usr/bin
+       cp out/rust/release/ztunnel ${{targets.destdir}}/usr/bin/ztunnel
+
+  - uses: strip
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: Compat package for ztunnel
+    pipeline:
+      - runs: |
+          mkdir -p "${{targets.contextdir}}"/usr/local/bin
+          ln -sf /usr/bin/ztunnel "${{targets.contextdir}}"/usr/local/bin/ztunnel
+    dependencies:
+      provides:
+        - ztunnel-compat=${{package.full-version}}
+
+update:
+  enabled: true
+  ignore-regex-patterns:
+    - '-rc'
+    - '-beta'
+  github:
+    identifier: istio/ztunnel
+    tag-filter-prefix: 1.24.
+    use-tag: true
+
+test:
```

Comment:
Anything more we can do in tests?

Reply:
the application tries to create `/var/run/ztunnel/ztunnel.sock` and fails. I was not able to add anything more in package level tests here because of same.